bridge/slack/slack.go

```diff
@@ -386,10 +390,11 @@ func (b *Bslack) uploadFile(msg *config.Message, channelID string) {
 		b.Log.Debugf("Adding file %s to cache at %s with timestamp", fi.Name, ts.String())
 		b.cache.Add("filename"+fi.Name, ts)
 		res, err := b.sc.UploadFile(slack.FileUploadParameters{
-			Reader:         bytes.NewReader(*fi.Data),
-			Filename:       fi.Name,
-			Channels:       []string{channelID},
-			InitialComment: fi.Comment,
+			Reader:          bytes.NewReader(*fi.Data),
+			Filename:        fi.Name,
+			Channels:        []string{channelID},
+			InitialComment:  fi.Comment,
```

Comment:
I actually assumed the empty message was to hint at who posted. Could we instead append/prepend the poster's username to the `InitialComment`? Since the file can be either uploaded or "created" (meaning that some content was posted to create the file within slack), then we could use a neutral phrase like:

> File from @username

> File from @username with comment: I am commenting on this file.

My understanding is that these are all the types of files that appear through this method: uploaded files, code snippets, shared docs. (all with and without initial comment to announce.)

![screen shot 2018-11-25 at 12 00 51 pm](https://user-images.githubusercontent.com/305339/48975375-c4ff3900-f0a9-11e8-81e5-1024769bc2b3.png)

Reply:
I would personally prefer the:
> File from @username

or

> File from @username with comment: I am commenting on this file.

over the other suggestion of simply adding the comment after a newline.

Reply:
Yeah, I added the newline because the Username by this point is often `patcon <CHANNEL>:` but sometimes <patcon> or whatever, depending on what RemoteNickFormat says. Newline skips having to care if we're doubling up the colon or whatever. I'm fine with whatever, but that's the rationale for the newline :)